resources/js/Pages/Search/Pending.vue

```diff
@@ -0,0 +1,80 @@
+<script lang="ts" setup>
+import ProgressSpinner from 'primevue/progressspinner';
+import axios from "axios";
+import route from "ziggy-js";
+import {Head, router, useForm, usePage} from "@inertiajs/vue3";
+import AppLayout from "@/Layouts/AppLayout.vue";
+import {onBeforeMount, onMounted, ref} from "vue";
+
+defineOptions({layout: AppLayout})
+
+const props = defineProps({
+    filename: String,
+    search: {type: Object, required: true},
+});
+
+var error = ref<boolean>(false)
+
+onBeforeMount(() => {
+    waitMode();
+});
+
+function waitMode() {
+    setTimeout(() => {
+        queryProcessed();
+    }, 5000);
+}
+
+function queryProcessed() {
+    axios.get(route('search.status', props.search.id))
+        .then(({data}) => {
+            console.log(data);
+            if (data.processed) {
+                router.visit(route('search.show', {
```

Comment:
This is working by chance, check the syntax on https://inertiajs.com/manual-visits : `router.visit(url, { method: 'post' })`

```suggestion
                router.visit(
                                route('search.show', { id: props.search.id }), // this is the url
                                { method: 'get', replace: true } // this is the configuration object
                )
```